netbox/utilities/views.py

```diff
@@ -39,6 +39,40 @@
 # Mixins
 #
 
+class ContentTypePermissionRequiredMixin(AccessMixin):
```

Comment:
Do you mind doing this bit since you likely already know exactly what you want here? I also want to make sure we cover the permission strings in the views today. For instance, the reports still use `view_reportresult`, etc.

Reply:
Yeah no problem, you can leave this as-is and I'll make adjustments in `develop-2.9` after merge.